Core/GameEngineDevice/Source/Win32Device/Common/Win32LocalFileSystem.cpp

```diff
@@ -139,7 +139,7 @@ void Win32LocalFileSystem::getFileListInDirectory(const AsciiString& currentDire
 
 	while (!done)	{
 		if (!(findData.dwFileAttributes & FILE_ATTRIBUTE_DIRECTORY) &&
-				(strcmp(findData.cFileName, ".") && strcmp(findData.cFileName, ".."))) {
+				(strcmp(findData.cFileName, ".") != 0 && strcmp(findData.cFileName, ".."))) {
```

Comment:
It somehow missed the second condition expression. It looks like clang tidy has some issue? It is a bit worrying to see such an obvious bug with the tool. Lowers the confidence in it.

Several times in this change.

Reply:
Ah, this is a design thing for clang-tidy to avoid conflicts when multiple checks would touch the same line. We're supposed to run it a second time - I'll do that now